<!DOCTYPE html PUBLIC "-//W3C//DTD HTML EXPERIMENTAL 970324//EN">
<html><head>
<meta http-equiv="content-type" content="text/html; charset=UTF-8"><meta name="viewport" content="width=device-width, initial-scale=1">
<meta name="GENERATOR" content="Adobe FrameMaker 5.5/HTML Export Filter"><link rel="STYLESHEET" href="Appendix%20III%20AutoDock%20File%20Formats_files/Using_AutoDock_305.css"><title> Appendix III: AutoDock File Formats</title></head>



<body bgcolor="#ffffff">
<ol>
<h1 class="Appndix">
<a name="pgfId=75765">
 </a>
AutoDock File Formats</h1>
</ol>
<p class="Body">
<a name="pgfId=30114">
 </a>
The formats will sometimes be given with notation such as '%d' to 
indicate a decimal integer; '%6.3f' for a floating point number with up 
to 6 characters and 3 digits after the decimal place; or '%-7s' for a 
left-justified string 7 characters wide. This notation is compatible 
with C, C++, awk/nawk/gawk, and with a slight modification, Python.</p>
<p class="Body">
<a name="pgfId=30147">
 </a>
The "" symbol is used to indicate one space.</p>
<p class="Body">
<a name="pgfId=30115">
 </a>
The type of an argument is described using the following style:</p>
<p class="Body">
<a name="pgfId=30116">
 </a>
<code class="Courier-Small">
&lt;string&gt;</code>
 = an alphanumeric string. In most cases, this is a valid filename;<br>
<code class="Courier-Small">
&lt;character&gt;</code>
 = a single letter;<br>
<code class="Courier-Small">
&lt;integer&gt;</code>
 = a decimal integer; <br>
<code class="Courier-Small">
&lt;positive_integer&gt;</code>
 = a decimal integer greater than zero; <br>
<code class="Courier-Small">
&lt;long_integer&gt;</code>
 = a decimal integer in the "long" range (depends on computer);<br>
<code class="Courier-Small">
&lt;float&gt;</code>
 = a floating point or real number.</p>
<div>
<ol>
<ol>
<h2 class="AppndixHeading1">
<a name="pgfId=29703">
 </a>
III  1. Protein Data Bank with Partial Charges: PDBQ</h2>
</ol>
</ol>
<p class="Body">
<a name="pgfId=34015">
 </a>
<strong class="Bold">
Extension: .pdbq</strong>
</p>
<p class="Body">
<a name="pgfId=34197">
 </a>
The name 'PDBQ' derives from 'PDB', the Protein DataBank, and 'Q', a 
common symbol for partial charge. As the name suggests, the PDBQ format 
is very similar to the PDB format for ATOM records, with a modification 
in columns 71-76 (counting the first column as 1, not 0) to carry the 
partial charge, as %6.3f. Thus, the format of the whole line is as 
follows:</p>
<pre class="CouriSmall"><a name="pgfId=34198"> </a>
"ATOM%5d%-4s%1s%-3s%1s%4d%1s%8.3f%8.3f%8.3f%6.2f%6.2f%4s%6.3f\n", atom_serial_num, atom_name, alt_loc, res_name, chain_id, res_num, ins_code, x, y, z,
occupancy, temp_factor, footnote, partial_charge
</pre>
<p class="Body">
<a name="pgfId=29747">
 </a>
In addition to this, there are various records (ROOT, TORSION and 
BRANCH, for example) in the ligand PDBQ file that specify which portion 
of the molecule is rigid and which is flexible.</p>
</div>
<div>
<ol>
<ol>
<h2 class="AppndixHeading1">
<a name="pgfId=29698">
 </a>
III  2. PDBQ with Solvation Parameters: PDBQS</h2>
</ol>
</ol>
<p class="Body">
<a name="pgfId=34016">
 </a>
<strong class="Bold">
Extension: .pdbqs</strong>
</p>
<p class="Body">
<a name="pgfId=34026">
 </a>
This format is derived from the PDBQ format, and is used to specify the 
atomic solvation parameters for the macromolecule, hence the "S". The 
format of the lines is:</p>
<pre class="CouriSmall"><a name="pgfId=29970"> </a>
"ATOM%5d%-4s%1s%-3s%1s%4d%1s%8.3f%8.3f%8.3f%6.2f%6.2f%4s%6.3f%8.2f%8.2f\n", atom_serial_num, atom_name, alt_loc, res_name, chain_id, res_num, ins_code, x, y, z,
occupancy, temp_factor, footnote, partial_charge,
atomic_fragmental_volume, atomic_solvation_parameter</pre>
<p class="Body">
<a name="pgfId=30079">
 </a>
The atomic fragmental volume and solvation parameters are derived from the method of Stouten <em class="Emphasis">
et al</em>
.<a href="#pgfId=30081" class="footnote">
1</a>
</p>
</div>
<div>
<ol>
<ol>
<h2 class="AppndixHeading1">
<a name="pgfId=10736">
 </a>
III  3. AutoGrid Grid Parameter File: GPF</h2>
</ol>
</ol>
<p class="Body">
<a name="pgfId=10737">
 </a>
<strong class="Bold">
Extension: .gpf</strong>
</p>
<p class="Body">
<a name="pgfId=34033">
 </a>
<strong class="Embolden">
The input file is often referred to as a "grid parameter file" or "GPF" 
for short. The scripts described in the appendices give these files the 
extension ".gpf". </strong>
In the grid parameter file, the user must specify the following spatial attributes of the grid maps:</p>
<p class="Body">
<a name="pgfId=10738">
 </a>
1.	the center of the grid map;</p>
<p class="Body">
<a name="pgfId=30094">
 </a>
2.	the number of grid points in each of the <em class="Emphasis">
x</em>
-, <em class="Emphasis">
y</em>
- and <em class="Emphasis">
z</em>
-directions; and</p>
<p class="Body">
<a name="pgfId=30095">
 </a>
3.	the separation or spacing of each grid point.</p>
<p class="Body">
<a name="pgfId=10739">
 </a>
In addition, the pairwise-atomic interaction energy parameters must be 
specified. The following lines are required for each ligand atom type, <em class="Emphasis">
Y</em>
:</p>
<p class="Body">
<a name="pgfId=10740">
 </a>
4.	the grid map filename for atom type <em class="Emphasis">
Y</em>
;</p>
<p class="Body">
<a name="pgfId=30093">
 </a>
5.	seven lines containing the non-bonded parameters for each pairwise-atomic interaction, in the following order: <em class="Emphasis">
Y</em>
-C, <em class="Emphasis">
Y</em>
-N, <em class="Emphasis">
Y</em>
-O, <em class="Emphasis">
Y</em>
-S, <em class="Emphasis">
Y</em>
-H, <em class="Emphasis">
Y</em>
-<em class="Emphasis">
X</em>
, (<em class="Emphasis">
X</em>
 is any other atom type) and <em class="Emphasis">
Y</em>
-<em class="Emphasis">
M</em>
 (<em class="Emphasis">
M</em>
 is a metal, say). </p>
<p class="Body">
<a name="pgfId=10744">
 </a>
Using coefficients <em class="Emphasis">
C</em>
<em class="Subscript">
n</em>
, <em class="Emphasis">
C</em>
<em class="Subscript">
m</em>
, <em class="Emphasis">
n </em>
and <em class="Emphasis">
m</em>
, the pairwise interaction energy, <em class="Emphasis">
V</em>
(<em class="Emphasis">
r</em>
) is given by:</p>
<div>
<img src="Appendix%20III%20AutoDock%20File%20Formats_files/Using_AutoDock_305-34.gif">
</div>
<p class="Body">
<a name="pgfId=10748">
 </a>
Alternatively, the user can specify reqm, e, <em class="Emphasis">
n </em>
and <em class="Emphasis">
m</em>
:</p>
<div>
<img src="Appendix%20III%20AutoDock%20File%20Formats_files/Using_AutoDock_305-35.gif">
</div>
<p class="Body">
<a name="pgfId=10749">
 </a>
This latter method of specification is more intuitive for the user, while <strong class="Bold">
AutoGrid</strong>
 handles the calculation of the coefficients. By default, the <em class="Emphasis">
Y</em>
-<em class="Emphasis">
X</em>
 and <em class="Emphasis">
Y</em>
-<em class="Emphasis">
M</em>
 lines are copies of the <em class="Emphasis">
Y</em>
-H line. But in some systems, such as receptors which consist of DNA/protein complexes, both sulphur <em class="Emphasis">
and</em>
 phosphorus can be present. In this scenario, the <em class="Emphasis">
Y</em>
-<em class="Emphasis">
X</em>
 line can be used for modeling interactions with receptor-phosphorus 
atoms. A very rough approximation for phosphorus parameters is to borrow
 those of carbon.</p>
<p class="Body">
<a name="pgfId=30130">
 </a>
The "elecmap" line in the grid parameter file is the filename of the 
electrostatic potential grid map. The following parameter, "dielectric",
 if negative, indicates that the distance-dependent dielectric function 
of Mehler and Solmajer<em class="Superscript">
3</em>
 will be used. If positive, however, the value of that number will be 
used as a constant dielectric. For example, if the value were 40.0, then
 a constant dielectric of 40 would be used.</p>
<p class="Body">
<a name="pgfId=30131">
 </a>
The <strong class="Embolden">
AutoGrid</strong>
 parameter file format is described below. </p>
<div>
<ol>
<ol>
<ol>
<h4 class="AppndixHeading2">
<a name="pgfId=30132">
 </a>
AutoGrid Keywords and Commands</h4>
</ol>
</ol>
</ol>
<p class="Keyword">
<a name="pgfId=10753">
 </a>
receptor &lt;string&gt;</p>
<p class="Body">
<a name="pgfId=10754">
 </a>
Macromolecule filename, in PDBQ format.</p>
<p class="Keyword">
<a name="pgfId=10755">
 </a>
gridfld &lt;string&gt;</p>
<p class="Body">
<a name="pgfId=10759">
 </a>
The grid field filename, which will be written in a format readable by <strong class="Bold">
AutoDock</strong>
 and AVS<a href="#pgfId=10758" class="footnote">
2</a>
. The filename extension <em class="Emphasis">
must</em>
 be `.fld'.</p>
<p class="Keyword">
<a name="pgfId=10760">
 </a>
npts &lt;integer&gt; &lt;integer&gt; &lt;integer&gt;</p>
<p class="Body">
<a name="pgfId=10761">
 </a>
Number of <em class="Emphasis">
x</em>
-, <em class="Emphasis">
y</em>
- and <em class="Emphasis">
z</em>
-grid points. Each <em class="Emphasis">
must</em>
 be an even integer number. When added to the central grid point, there 
will be an odd number of points in each dimension. The number of <em class="Emphasis">
x</em>
-, <em class="Emphasis">
y</em>
- and <em class="Emphasis">
z</em>
-grid points need not be equal.</p>
<p class="Keyword">
<a name="pgfId=10762">
 </a>
spacing &lt;float&gt;</p>
<p class="Body">
<a name="pgfId=10763">
 </a>
The grid point spacing, in Å (see the diagram on page 8). Grid points 
must be uniformly spaced in AutoDock: this value is used in each 
dimension.</p>
<p class="Keyword">
<a name="pgfId=10764">
 </a>
gridcenter &lt;float&gt; &lt;float&gt; &lt;float&gt;</p>
<p class="Keyword">
<a name="pgfId=10765">
 </a>
gridcenter auto</p>
<p class="Body">
<a name="pgfId=10766">
 </a>
The user can explicitly define the center of the grid maps, respectively the <em class="Emphasis">
x</em>
, <em class="Emphasis">
y</em>
 and <em class="Emphasis">
z</em>
 coordinates of the center of the grid maps (units: Å, Å, Å.) Or the keyword "auto" can be given, in which case <strong class="Embolden">
AutoGrid</strong>
 will center the grid maps on the center of mass of the macromolecule.</p>
<p class="Keyword">
<a name="pgfId=10767">
 </a>
types &lt;string&gt;</p>
<p class="Body">
<a name="pgfId=10768">
 </a>
1-letter names of the atom types present in the ligand; <em class="Emphasis">
e.g.</em>
 if there are carbons, nitrogens, oxygens and hydrogens, then this line will be "CNOH"; there are no delimiters.</p>
<p class="Keyword">
<a name="pgfId=10769">
 </a>
smooth &lt;float&gt;</p>
<p class="Body">
<a name="pgfId=41432">
 </a>
This is always 0.5Å, when using the AutoDock 3.0 free energy function. 
It is used to smooth the pairwise atomic affinity potentials (both van 
der Waals and hydrogen bonds). See the Theory section for more details.</p>
<p class="Keyword">
<a name="pgfId=41438">
 </a>
map &lt;string&gt;</p>
<p class="Body">
<a name="pgfId=10770">
 </a>
Filename of the grid map, for ligand atom type Y; the extension is usually ".map".</p>
<p class="Keyword">
<a name="pgfId=10771">
 </a>
nbp_coeffs &lt;float&gt; &lt;float&gt; &lt;integer&gt; &lt;integer&gt;</p>
<p class="Body">
<a name="pgfId=10772">
 </a>
Either "nbp_coeffs" or "nbp_r_eps" keywords can be used to define 
Lennard-Jones or hydrogen bond interaction energy parameters. The 
keyword "nbp_coeffs" specifies coefficients and exponents, in the order <em class="Emphasis">
"C</em>
<em class="Subscript">
n</em>
 <em class="Emphasis">
C</em>
<em class="Subscript">
m</em>
 <em class="Emphasis">
n</em>
 <em class="Emphasis">
m</em>
", delimited by spaces; <em class="Emphasis">
n</em>
 and <em class="Emphasis">
m</em>
 are integer exponents. The units of <em class="Emphasis">
C</em>
<em class="Subscript">
n</em>
 and <em class="Emphasis">
C</em>
<em class="Subscript">
m</em>
 must be kcal mol<em class="Superscript">
-1</em>
 Å<em class="Superscript">
n</em>
 and kcal mol<em class="Superscript">
-1</em>
 Å<em class="Superscript">
m</em>
 respectively; <em class="Emphasis">
n</em>
 and <em class="Emphasis">
m</em>
 have no units. </p>
<p class="Keyword">
<a name="pgfId=10773">
 </a>
nbp_r_eps &lt;float&gt; &lt;float&gt; &lt;integer&gt; &lt;integer&gt;</p>
<p class="Body">
<a name="pgfId=10774">
 </a>
Alternatively, the user can employ "nbp_r_eps" to specify the 
equilibrium distance and well depth, epsilon, for the atom pair. The 
equilibrium separation has units of Å and the well depth, epsilon, units
 of kcal mol<em class="Superscript">
-1</em>
. The integer exponents <em class="Emphasis">
n</em>
 and <em class="Emphasis">
m</em>
 must be specified too.</p>
<p class="Body">
<a name="pgfId=10775">
 </a>
In either case, the order of the parameters must be: <em class="Emphasis">
Y</em>
-C, <em class="Emphasis">
Y</em>
-N, <em class="Emphasis">
Y</em>
-O, <em class="Emphasis">
Y</em>
-S, <em class="Emphasis">
Y</em>
-H, <em class="Emphasis">
Y</em>
-X, and <em class="Emphasis">
Y</em>
-M. Repeat 1 "map" line and the 7 "nbp_coeffs"or "nbp_r_eps" lines, for each atom type, <em class="Emphasis">
Y</em>
, present in the ligand being docked.</p>
<p class="Keyword">
<a name="pgfId=41439">
 </a>
sol_par &lt;float&gt; &lt;float&gt;</p>
<p class="Body">
<a name="pgfId=41440">
 </a>
This is used to define the atomic fragmental volume and solvation 
parameters, and should not be changed from the Stouten values used to 
calibrate the AutoDock 3.0 free energy function.</p>
<p class="Keyword">
<a name="pgfId=41441">
 </a>
constant &lt;float&gt;</p>
<p class="Body">
<a name="pgfId=41442">
 </a>
This is added to all the values in a grid map, and is only set to a 
non-zero, positive number for hydrogen bonding maps. This value is 
essentially the penalty for un-formed hydrogen bonds in the complex.</p>
<p class="Keyword">
<a name="pgfId=10776">
 </a>
elecmap &lt;string&gt;</p>
<p class="Body">
<a name="pgfId=10777">
 </a>
Filename for the electrostatic potential energy grid map to be created; filename extension `.map'. </p>
<p class="Keyword">
<a name="pgfId=10778">
 </a>
dielectric &lt;float&gt;</p>
<p class="Body">
<a name="pgfId=10779">
 </a>
Dielectric function flag: if negative, <strong class="Bold">
AutoGrid</strong>
 will use <em class="Emphasis">
distance-dependent</em>
 dielectric of Mehler and Solmajer<em class="Superscript">
3</em>
; if the float is positive, <strong class="Bold">
AutoGrid</strong>
 will use this value as the dielectric constant.</p>
<p class="Keyword">
<a name="pgfId=10780">
 </a>
fmap &lt;string&gt;</p>
<p class="Body">
<a name="pgfId=10784">
 </a>
<em class="Emphasis">
(Optional.) </em>
Filename for the so-called "floating" grid map<a href="#pgfId=10783" class="footnote">
3</a>
; filename extension `.map'. In such floating grids, the scalar at each 
grid point is the distance to the nearest atom in the receptor. These 
values could be used to guide the docking ligand towards the receptor's 
surface, thus avoiding non-interesting, empty regions. </p>
</div>
</div>
<div>
<ol>
<ol>
<h2 class="AppndixHeading1">
<a name="pgfId=34041">
 </a>
III  4. Grid Map File</h2>
</ol>
</ol>
<p class="Body">
<a name="pgfId=34042">
 </a>
<strong class="Bold">
Extension: .map</strong>
</p>
<p class="Body">
<a name="pgfId=34043">
 </a>
The first six lines of each grid map hold header information which 
describe the spatial features of the maps and the files used or created.
 These headers are checked by <strong class="Bold">
AutoDock</strong>
 to ensure that they are appropriate for the requested docking. The 
remainder of the file contains grid point energies, written as floating 
point numbers, one per line. They are ordered according to the nested 
loops z( y( x ) ). A sample header from a grid map is shown below:</p>
<p class="Body">
<a name="pgfId=11329">
 </a>
______________________________________________________________________________</p>
<pre class="CouriSmall"><a name="pgfId=11330"> </a>
GRID_PARAMETER_FILE vac1.nbc.gpf</pre>
<pre class="CouriSmall"><a name="pgfId=11331"> </a>
GRID_DATA_FILE 4phv.nbc_maps.fld</pre>
<pre class="CouriSmall"><a name="pgfId=11332"> </a>
MACROMOLECULE 4phv.new.pdbq</pre>
<pre class="CouriSmall"><a name="pgfId=11333"> </a>
SPACING 0.375</pre>
<pre class="CouriSmall"><a name="pgfId=11334"> </a>
NELEMENTS 50 50 80</pre>
<pre class="CouriSmall"><a name="pgfId=11335"> </a>
CENTER -0.026 4.353 -0.038</pre>
<pre class="CouriSmall"><a name="pgfId=11336"> </a>
125.095596</pre>
<pre class="CouriSmall"><a name="pgfId=11337"> </a>
123.634560</pre>
<pre class="CouriSmall"><a name="pgfId=11338"> </a>
116.724602</pre>
<pre class="CouriSmall"><a name="pgfId=11339"> </a>
108.233879</pre>
<pre class="CouriSmall"><a name="pgfId=11340"> </a>
:</pre>
<p class="Body">
<a name="pgfId=11341">
 </a>
______________________________________________________________________________</p>
</div>
<div>
<ol>
<ol>
<h2 class="AppndixHeading1">
<a name="pgfId=10734">
 </a>
III  5. Grid Map Field File</h2>
</ol>
</ol>
<p class="Body">
<a name="pgfId=34046">
 </a>
<strong class="Bold">
Extension: .maps.fld</strong>
</p>
<p class="Body">
<a name="pgfId=36545">
 </a>
This is essentially two files in one. It is both an AVS field file, and and <strong class="Bold">
AutoDock</strong>
 input file with <strong class="Bold">
AutoDock</strong>
-specific information `hidden' from AVS in the comments at the head of the file. <strong class="Bold">
AutoDock</strong>
 uses this file to check that all the maps it reads in are compatible 
with one-another and itself. For example, in this file, the grid spacing
 is 0.375 Angstroms, there are 60 intervals in each dimension, the grid 
is centered near (46,44,14), it was calculated around the macromolecule `<code class="Courier-Small">
2cpp.pdbqs</code>
', and the <strong class="Bold">
AutoGrid</strong>
 parameter file used to create this and the maps was `<code class="Courier-Small">
2cpp.gpf</code>
'. This file also points to a second file, `<code class="Courier-Small">
2cpp.maps.xyz</code>
', which contains the minimum and maximum extents of the grid box in each dimension, <em class="Emphasis">
x, y,</em>
 and <em class="Emphasis">
z</em>
. Finally, it lists the grid map files that were calculated by <strong class="Bold">
AutoGrid</strong>
, here `<code class="Courier-Small">
2cpp.C.map</code>
', `<code class="Courier-Small">
2cpp.O.map</code>
' and `<code class="Courier-Small">
2cpp.e.map</code>
'.</p>
<p class="Body">
<a name="pgfId=36551">
 </a>
______________________________________________________________________________</p>
<pre class="CouriSmall"><a name="pgfId=36562"> </a>
# AVS field file</pre>
<pre class="CouriSmall"><a name="pgfId=36569"> </a>
#</pre>
<pre class="CouriSmall"><a name="pgfId=36570"> </a>
# AutoDock Atomic Affinity and Electrostatic Grids</pre>
<pre class="CouriSmall"><a name="pgfId=36571"> </a>
#</pre>
<pre class="CouriSmall"><a name="pgfId=36572"> </a>
# Created by autogrid3.</pre>
<pre class="CouriSmall"><a name="pgfId=36573"> </a>
#</pre>
<pre class="CouriSmall"><a name="pgfId=36574"> </a>
#SPACING 0.375</pre>
<pre class="CouriSmall"><a name="pgfId=36575"> </a>
#NELEMENTS 60 60 60</pre>
<pre class="CouriSmall"><a name="pgfId=36576"> </a>
#CENTER 46.508 44.528 14.647</pre>
<pre class="CouriSmall"><a name="pgfId=36577"> </a>
#MACROMOLECULE 2cpp.pdbqs</pre>
<pre class="CouriSmall"><a name="pgfId=36578"> </a>
#GRID_PARAMETER_FILE 2cpp.gpf</pre>
<pre class="CouriSmall"><a name="pgfId=36579"> </a>
#</pre>
<pre class="CouriSmall"><a name="pgfId=36580"> </a>
ndim=3                  # number of dimensions in the field</pre>
<pre class="CouriSmall"><a name="pgfId=36581"> </a>
dim1=61                 # number of x-elements</pre>
<pre class="CouriSmall"><a name="pgfId=36582"> </a>
dim2=61                 # number of y-elements</pre>
<pre class="CouriSmall"><a name="pgfId=36583"> </a>
dim3=61                 # number of z-elements</pre>
<pre class="CouriSmall"><a name="pgfId=36584"> </a>
nspace=3                # number of physical coordinates per point</pre>
<pre class="CouriSmall"><a name="pgfId=36585"> </a>
veclen=3                # number of affinity values at each point</pre>
<pre class="CouriSmall"><a name="pgfId=36586"> </a>
data=float              # data type (byte, integer, float, double)</pre>
<pre class="CouriSmall"><a name="pgfId=36587"> </a>
field=uniform           # field type (uniform, rectilinear, irregular)</pre>
<pre class="CouriSmall"><a name="pgfId=36588"> </a>
coord 1 file=2cpp.maps.xyz filetype=ascii offset=0</pre>
<pre class="CouriSmall"><a name="pgfId=36589"> </a>
coord 2 file=2cpp.maps.xyz filetype=ascii offset=2</pre>
<pre class="CouriSmall"><a name="pgfId=36590"> </a>
coord 3 file=2cpp.maps.xyz filetype=ascii offset=4</pre>
<pre class="CouriSmall"><a name="pgfId=36591"> </a>
label=C-affinity        # component label for variable 1</pre>
<pre class="CouriSmall"><a name="pgfId=36592"> </a>
label=O-affinity        # component label for variable 2</pre>
<pre class="CouriSmall"><a name="pgfId=36593"> </a>
label=Electrostatics    # component label for variable 2</pre>
<pre class="CouriSmall"><a name="pgfId=36594"> </a>
#</pre>
<pre class="CouriSmall"><a name="pgfId=36595"> </a>
# location of affinity grid files and how to read them</pre>
<pre class="CouriSmall"><a name="pgfId=36596"> </a>
#</pre>
<pre class="CouriSmall"><a name="pgfId=36597"> </a>
variable 1 file=2cpp.C.map filetype=ascii skip=6</pre>
<pre class="CouriSmall"><a name="pgfId=36598"> </a>
variable 2 file=2cpp.O.map filetype=ascii skip=6</pre>
<pre class="CouriSmall"><a name="pgfId=36599"> </a>
variable 3 file=2cpp.e.map filetype=ascii skip=6</pre>
<p class="Body">
<a name="pgfId=36546">
 </a>
______________________________________________________________________________</p>
</div>
<div>
<ol>
<ol>
<h2 class="AppndixHeading1">
<a name="pgfId=36547">
 </a>
III  6. AutoDock Docking Parameter File: DPF</h2>
</ol>
</ol>
<p class="Body">
<a name="pgfId=10840">
 </a>
<strong class="Bold">
Extension: .dpf</strong>
</p>
<p class="Body">
<a name="pgfId=34070">
 </a>
<strong class="Bold">
AutoDock 3.0</strong>
<strong class="Embolden">
 has an interface based on keywords. This is intended to make it easier 
for the user to set up and control a docking job, and for the programmer
 to add new commands and functionality. The input file is often referred
 to as a "docking parameter file" or "DPF" for short. The scripts 
described in the appendices give these files the extension ".dpf".</strong>
</p>
<p class="Body">
<a name="pgfId=30121">
 </a>
All delimiters where needed are white spaces. Default values, where 
applicable, are given in square brackets [thus]. A comment must be 
prefixed by the "<code class="Courier">
#</code>
" symbol, and can be placed at the end of a parameter line, or on a line of its own. </p>
<p class="Body">
<a name="pgfId=10843">
 </a>
Although ideally it should be possible to give these keywords in any 
order, not every possible combination has been tested, so it would be 
wise to stick to the following order.</p>
<div>
<ol>
<ol>
<ol>
<h4 class="AppndixHeading2">
<a name="pgfId=10844">
 </a>
Command to set the seed for the random number generator</h4>
</ol>
</ol>
</ol>
<p class="Keyword">
<a name="pgfId=10845">
 </a>
seed &lt;long_integer&gt;</p>
<p class="Keyword">
<a name="pgfId=10846">
 </a>
seed time</p>
<p class="Keyword">
<a name="pgfId=10847">
 </a>
seed pid</p>
<p class="Keyword">
<a name="pgfId=10848">
 </a>
seed &lt;long_integer&gt; &lt;long_integer&gt;</p>
<p class="Keyword">
<a name="pgfId=10849">
 </a>
seed time &lt;long_integer&gt;</p>
<p class="Keyword">
<a name="pgfId=10850">
 </a>
seed &lt;long_integer&gt; time</p>
<p class="Keyword">
<a name="pgfId=10851">
 </a>
seed time pid</p>
<p class="Keyword">
<a name="pgfId=10852">
 </a>
seed pid &lt;long_integer&gt;</p>
<p class="Keyword">
<a name="pgfId=10853">
 </a>
seed &lt;long_integer&gt; pid</p>
<p class="Keyword">
<a name="pgfId=10854">
 </a>
seed pid time</p>
<p class="Body">
<a name="pgfId=10855">
 </a>
There are two possible random number generator libraries. One is the 
system's own implementations, and the second is the platform-independent
 library from the University of Texas Biomedical School. If the user 
gives just one argument to "<code class="Courier-Small">
seed</code>
", then AutoDock will use the system's implementation of the random 
number generator and corresponding system seed call. On most platforms, 
these are "drand48" and "srand48". The platform-independent library, 
however, requires two seed values. Giving two arguments to "seed" tells 
AutoDock to use the platform-independent library for random number 
generation.</p>
<p class="Body">
<a name="pgfId=10856">
 </a>
The random-number generator (RNG) for each docking job can be `seeded' 
with either a user-defined, a time-dependent, or process-ID-dependent 
seed. These two seeds can be any combination of explicit long integers, 
the keyword "<code class="Courier-Small">
time</code>
" or the keyword "<code class="Courier-Small">
pid</code>
".  When two arguments to seed are given, the portable RNG is used; when
 one is given, the built-in RNG (usually the "drand48" C-function) is 
used.  The portable RNG is required for the genetic algorithm and the 
Solis and Wets routines.  The portable RNG cannot be used with the 
simulated annealing routine: this needs just one seed parameter. The 
keyword, "time" gives the number of seconds since the epoch. The epoch 
is referenced to 00:00:00 CUT (Coordinated Universal Time) 1 Jan 1970. 
The "pid" gives the UNIX process ID of the currently executing AutoDock 
process, which is reading this parameter file.</p>
</div>
<div>
<ol>
<ol>
<ol>
<h4 class="AppndixHeading2">
<a name="pgfId=10857">
 </a>
Parameters defining the grid maps to be used</h4>
</ol>
</ol>
</ol>
<p class="Keyword">
<a name="pgfId=10858">
 </a>
types &lt;string&gt;</p>
<p class="Body">
<a name="pgfId=10859">
 </a>
Atom names for all atom types present in ligand. Each must be a single 
character, and only one of: C, N, O, S, H, X, or M. The maximum number 
of characters allowed in this line is ATOM_MAPS, which is defined in the
 "autodock.h" include file. Do not use any spaces to delimit the types: 
they are not needed.</p>
<p class="Keyword">
<a name="pgfId=10860">
 </a>
fld &lt;string&gt;</p>
<p class="Body">
<a name="pgfId=10861">
 </a>
Grid data field file created by <strong class="Bold">
AutoGrid</strong>
 and readable by <strong class="Bold">
AVS</strong>
 (must have the extension ".fld").</p>
<p class="Keyword">
<a name="pgfId=10862">
 </a>
map &lt;string&gt;</p>
<p class="Body">
<a name="pgfId=10863">
 </a>
Filename for the first <strong class="Bold">
AutoGrid </strong>
affinity grid map of the 1st atom type. This keyword plus filename must 
be repeated for all atom types in the order specifed by the "<code class="Courier-Small">
types</code>
" command. In all map files a 6-line header is required, and energies must be ordered according to the nested loops z( y( x ) ).</p>
<p class="Keyword">
<a name="pgfId=10864">
 </a>
map &lt;string&gt;</p>
<p class="Body">
<a name="pgfId=10865">
 </a>
Filename for the electrostatics grid map. 6-line header required, and 
energies must be ordered according to the nested loops z( y( x ) ).</p>
</div>
<div>
<ol>
<ol>
<ol>
<h4 class="AppndixHeading2">
<a name="pgfId=10866">
 </a>
Parameters defining the ligand and its initial state</h4>
</ol>
</ol>
</ol>
<p class="Keyword">
<a name="pgfId=10867">
 </a>
move &lt;string&gt;</p>
<p class="Body">
<a name="pgfId=10868">
 </a>
Filename for the ligand to be docked. This contains most importantly, 
atom names, xyz-coordinates, anb partial atomic charges in PDBQ format. 
(Filename extension should be ".pdbq").</p>
<p class="Keyword">
<a name="pgfId=10869">
 </a>
about &lt;float&gt; &lt;float&gt; &lt;float&gt;</p>
<p class="Body">
<a name="pgfId=10870">
 </a>
Use this keyword to specify the center of the ligand, <em class="Emphasis">
about</em>
 which rotations will be made. (The coordinate frame of reference is 
that of the ligand PDBQ file.) Usually the rotation center of the ligand
 is the mean x,y,z-coordinates of the molecule. Inside <strong class="Bold">
AutoDock</strong>
, the "<code class="Courier-Small">
about</code>
" xyz-coordinates are <em class="Emphasis">
subtracted</em>
 from each atom's coordinates in the input PDBQ file. So internally, the
 ligand's coordinates become centered at the origin. Units: Å, Å, Å.</p>
<p class="Keyword">
<a name="pgfId=10871">
 </a>
tran0 &lt;float&gt; &lt;float&gt; &lt;float&gt;</p>
<p class="Keyword">
<a name="pgfId=10872">
 </a>
tran0 random</p>
<p class="Body">
<a name="pgfId=10873">
 </a>
Initial coordinates for the center of the ligand, in the same frame of 
reference as the receptor's grid maps. The ligand, which has been 
internally centered using the "about" coordinates, has the 
xyz-coordinates of the initial translation "<code class="Courier-Small">
tran0 x y z</code>
" <em class="Emphasis">
added</em>
 on. Every run starts the ligand from this location.</p>
<p class="Body">
<a name="pgfId=10874">
 </a>
Alternatively, the user can just give the keyword "random" and <strong class="Bold">
AutoDock</strong>
 will pick random initial coordinates instead.</p>
<p class="Body">
<a name="pgfId=10875">
 </a>
If there are multiple runs defined in this file, using the keyword "runs", then each new run will begin at this same location.</p>
<p class="Body">
<a name="pgfId=10876">
 </a>
The user <em class="Emphasis">
must</em>
 specify the absolute <em class="Emphasis">
starting</em>
 coordinates for the ligand, used to start each run. The user should 
ensure that the ligand, when translated to these coordinates, still fits
 within the volume of the grid maps. If there are some atoms which lie 
outside the grid volume, then AutoDock will automatically correct this, 
until the ligand is pulled completely within the volume of the grids. 
(This is necessary in order to obtain complete information about the 
energy of the initial state of the system.) The user will be notified of
 any such changes to the initial translation by AutoDock. (Units: Å, Å, 
Å.)</p>
<p class="Keyword">
<a name="pgfId=10877">
 </a>
quat0 &lt;float&gt; &lt;float&gt; &lt;float&gt; &lt;float&gt;</p>
<p class="Keyword">
<a name="pgfId=10878">
 </a>
quat0 random</p>
<p class="Body">
<a name="pgfId=10879">
 </a>
[1, 0, 0, 0°]<br>
Respectively: Qx, Qy, Qz<em class="Emphasis">
, Q</em>
<em class="Subscript">
w. </em>
Initial quaternion (applied to ligand) ­- Qx, Qy, Qz define the unit vector of the direction of rigid body rotation, and<em class="Emphasis">
 Q</em>
<em class="Subscript">
w </em>
defines the angle of rotation about this unit vector, in ° . (Units: none,none,none, °.)</p>
<div>
<img src="Appendix%20III%20AutoDock%20File%20Formats_files/Using_AutoDock_305-36.gif">
</div>
<p class="Body">
<a name="pgfId=10893">
 </a>
Alternatively, the user can just give the keyword "random" and <strong class="Bold">
AutoDock</strong>
 will pick a random unit vector and a random rotation (between 0° and 
360°) about this unit vector. Each run will begin at this same random 
rigid body rotation.</p>
<p class="Keyword">
<a name="pgfId=10894">
 </a>
ndihe &lt;integer&gt;</p>
<p class="Body">
<a name="pgfId=10895">
 </a>
Number of dihedrals or rotatable bonds in the ligand. This may be 
specifed only if rotatable bonds have been defined using ROOT, BRANCH, 
TORS <em class="Emphasis">
etc</em>
. keywords in the PDBQ<code class="Courier">
 </code>
file named on the "<code class="Courier">
move</code>
" line. The number supplied to this command must agree with the number 
of torsions defined in this ligand PDBQ file. If this keyword is used, 
then the next keyword, <code class="Courier-Bold">
dihe0</code>
, must also be specified. Note that if <code class="Courier-Bold">
ndihe</code>
 and <code class="Courier-Bold">
dihe0</code>
 are not specified and there are defined torsions in the ligand PDBQ file, <strong class="Bold">
AutoDock</strong>
 assumes that the <em class="Emphasis">
chi</em>
<em class="Subscript">
1</em>
,<em class="Emphasis">
 chi</em>
<em class="Subscript">
2</em>
,<em class="Emphasis">
 chi</em>
<em class="Subscript">
3</em>
, <em class="Emphasis">
etc.</em>
 are all zero, and does not change the initial ligand torsion angles. (See also "torsdof" below).</p>
<p class="Keyword">
<a name="pgfId=10896">
 </a>
dihe0 &lt;float&gt; ...</p>
<p class="Body">
<a name="pgfId=10897">
 </a>
Initial <strong class="Bold">
relative</strong>
 dihedral angles; there must be <code class="Courier">
ndihe</code>
 floating point numbers specified on this line. Each  value specified 
here will be added to the corresponding torsion angle in the input PDBQ 
file, at the start of each run. Torsion angles are only specified by two
 atoms, so the definition of rotations is relative. Units: °.</p>
</div>
<div>
<ol>
<ol>
<ol>
<h4 class="AppndixHeading2">
<a name="pgfId=33955">
 </a>
Parameters defining ligand step sizes</h4>
</ol>
</ol>
</ol>
<p class="Keyword">
<a name="pgfId=10899">
 </a>
tstep &lt;float&gt;</p>
<p class="Keyword">
<a name="pgfId=10900">
 </a>
tstep &lt;float&gt; &lt;float&gt;</p>
<p class="Body">
<a name="pgfId=10901">
 </a>
[2.0 Å]<br>
The first form, with one argument, defines the maximum translation jump 
for the first cycle that the ligand may make in one simulated annealing 
step. When "trnrf" is less than 1, the reduction factor is multiplied 
with the tstep at the end of each cycle, to give the new value for the 
next cycle. The second form allows the user to specify the value for the
 first cycle and the last cycle: AutoDock then calculates the reduction 
factor that satisfies these constraints. 	Units: Å. </p>
<p class="Keyword">
<a name="pgfId=10902">
 </a>
qstep &lt;float&gt;</p>
<p class="Body">
<a name="pgfId=10903">
 </a>
[50.0°]<br>
Maximum orientation step size for the angular component, <em class="Emphasis">
w</em>
, of quaternion. Units: °.</p>
<p class="Keyword">
<a name="pgfId=10904">
 </a>
dstep &lt;float&gt;</p>
<p class="Keyword">
<a name="pgfId=31233">
 </a>
[50.0°]</p>
<p class="Body">
<a name="pgfId=10905">
 </a>
Maximum dihedral (torsion) step size. Units: °.</p>
</div>
<div>
<ol>
<ol>
<ol>
<h4 class="AppndixHeading2">
<a name="pgfId=10906">
 </a>
Parameters defining optional ligand torsion constraints</h4>
</ol>
</ol>
</ol>
<p class="Keyword">
<a name="pgfId=10907">
 </a>
barrier &lt;float&gt;</p>
<p class="Body">
<a name="pgfId=10908">
 </a>
[10000.0]<br>
<em class="Emphasis">
(Optional)</em>
 This defines the energy-barrier height applied to constrained torsions.
 When the torsion is at a preferred angle, there is no torsion penalty: 
this torsion's energy is zero. If the torsion angle falls within a 
disallowed zone, however, it can contribute up to the full barrier 
energy. Since the torsion-energy profiles are stored internally as 
arrays of type `unsigned short', only positive integers between 0 and 
65535 are allowed.</p>
<p class="Keyword">
<a name="pgfId=10909">
 </a>
gausstorcon &lt;integer&gt; &lt;float&gt; &lt;float&gt;</p>
<p class="Body">
<a name="pgfId=10910">
 </a>
<em class="Emphasis">
(Optional) </em>
Adds a constraint to a torsion. The torsion number is identified by an 
integer. This identifier comes from the list at the top of the 
AutoTors-generated input ligand PDBQ file (on the REMARK lines). An 
energy profile will be calculated for this torsion. An inverted Gaussian
 bell curve is added for each new constraint. To completely specify each
 Gaussian, two floating point numbers are needed: the <em class="Emphasis">
preferred angle</em>
 and the <em class="Emphasis">
half-width</em>
 respectively (both in degrees). Note that the preferred angle should be specified in the range -180<em class="Emphasis">
°</em>
 to +180<em class="Emphasis">
°</em>
; numbers outside this range will be wrapped back into this range. This angle, c, is <em class="Emphasis">
relative</em>
 to the original torsion angle in the input structure. The <em class="Emphasis">
half-width</em>
 is the difference between the two angles at which the energy is half 
the barrier (B/2 in the diagram above). The smaller the half-width, the 
tighter the constraint. </p>
<div>
<img src="Appendix%20III%20AutoDock%20File%20Formats_files/Using_AutoDock_305-37.gif">
</div>
<p class="Body">
<a name="pgfId=10960">
 </a>
If you wish to constrain to absolute-valued torsion angles, it will be 
necessary to zero the initial torsion angles in the ligand, before input
 to <strong class="Bold">
AutoTors</strong>
. The problem arises from the ambiguous 2-atom definition of the rotatable bond <em class="Emphasis">
B-C</em>
. To identify a torsion angle unambiguously, 4 atoms must be specified: <em class="Emphasis">
A-B-C-D</em>
: </p>
<div>
<img src="Appendix%20III%20AutoDock%20File%20Formats_files/Using_AutoDock_305-38.gif">
</div>
<p class="Body">
<a name="pgfId=11005">
 </a>
The sign convention for torsion angles which we use is anti-clockwise 
(counter-clockwise) are positive angles, clockwise negative. In the 
above diagram, looking down the bond <em class="Emphasis">
B-C</em>
, the dihedral angle <em class="Emphasis">
A-B-C-D</em>
 would be positive. </p>
<p class="Body">
<a name="pgfId=11006">
 </a>
There is no limit to the number of constraints that can be added to a 
given torsion. Each new torsion-constraint energy profile is combined 
with the pre-existing one by selecting the minimum energy of either the 
new or the existing profiles.</p>
<p class="Keyword">
<a name="pgfId=11007">
 </a>
showtorpen</p>
<p class="Body">
<a name="pgfId=11008">
 </a>
<em class="Emphasis">
(Optional) (Use only with "gausstorcon") </em>
This switches on the storage and subsequent output of torsion energies. 
During each energy evaluation, the penalty energy for each constrained 
torsion, as specified by the "gausstorcon" command, will be stored in an
 array. At the end of each run, the final docked conformation's state 
variables are output, but with this command, the penalty energy for each
 torsion will be printed alongside its torsion angle.</p>
<p class="Keyword">
<a name="pgfId=11009">
 </a>
hardtorcon &lt;integer&gt; &lt;float&gt; &lt;float&gt;</p>
<p class="Body">
<a name="pgfId=11010">
 </a>
<em class="Emphasis">
(Optional) </em>
This command also adds a torsion constraint to the &lt;integer&gt;-th 
torsion, as numbered in the AutoTors-generated REMARKs. The first float 
defines the <em class="Emphasis">
preferred relative angle</em>
, and the second specifies the <em class="Emphasis">
full width</em>
 of the allowed range of torsion angles (both in degrees). This type of 
torsion constraint is "hard" because the torsion is never allowed to 
take values beyond the range defined. For example, "	hardtorcon 3 60. 
10." would constrain the third torsion to values between 55° and 65°.</p>
</div>
<div>
<ol>
<ol>
<ol>
<h4 class="AppndixHeading2">
<a name="pgfId=11573">
 </a>
Parameter affecting torsional free energy</h4>
</ol>
</ol>
</ol>
<p class="Keyword">
<a name="pgfId=11011">
 </a>
torsdof &lt;integer&gt; &lt;float&gt;</p>
<p class="Body">
<a name="pgfId=11012">
 </a>
[0, 0.3113]<br>
This specifies respectively the number and the coefficient of the 
torsional degrees of freedom (DOF) for the estimation of the change in 
free energy upon binding, D<em class="Emphasis">
G</em>
<em class="Subscript">
binding</em>
. For the purposes of AutoDock 3.0, the number of torsional DOF is the 
number of rotatable bonds in the ligand, excluding any torsions that 
rotate one or more hydrogen atoms, <em class="Emphasis">
e.g.</em>
 hydroxyls, amines, methyls. By default, the coefficient is 0.3113 kcal mol<em class="Superscript">
-1</em>
, although the user can override this as necessary. (Units: none; kcal mol<em class="Superscript">
-1</em>
).</p>
</div>
<div>
<ol>
<ol>
<ol>
<h4 class="AppndixHeading2">
<a name="pgfId=11013">
 </a>
Parameters for ligand internal energies</h4>
</ol>
</ol>
</ol>
<p class="Keyword">
<a name="pgfId=11014">
 </a>
intnbp_coeffs &lt;float&gt; &lt;float&gt; &lt;integer&gt; &lt;integer&gt;</p>
<p class="Body">
<a name="pgfId=11015">
 </a>
Respectively: Cn; Cm; n; m. This command specifies the internal pairwise
 non-bonded energy parameters for flexible ligands, where:</p>
<div>
<img src="Appendix%20III%20AutoDock%20File%20Formats_files/Using_AutoDock_305-39.gif">
</div>
<p class="Body">
<a name="pgfId=11019">
 </a>
These parameters are needed even if no rotatable bonds were defined in 
the ligand-PDBQ file. They are only used in the internal energy 
calculations for the ligand and must be consistent with those used in 
calculating the grid maps. (Units: kcal mol<em class="Superscript">
-1</em>
 Å<em class="Superscript">
n</em>
; kcal mol<em class="Superscript">
-1</em>
 Å<em class="Superscript">
m</em>
; none; none, respectively).</p>
<p class="Keyword">
<a name="pgfId=11020">
 </a>
intnbp_r_eps &lt;float&gt; &lt;float&gt; &lt;integer&gt; &lt;integer&gt;</p>
<p class="Body">
<a name="pgfId=11021">
 </a>
Respectively: reqm; e; n; m, This command is an alternative way of 
specifying the internal pairwise non-bonded energy parameters for 
flexible ligands, where <strong class="Bold">
AutoDock</strong>
 calculates the pairwise atomic potential using:</p>
<div>
<img src="Appendix%20III%20AutoDock%20File%20Formats_files/Using_AutoDock_305-40.gif">
</div>
<p class="Body">
<a name="pgfId=11025">
 </a>
The first two arguments specify the equilibrium distance and well depth,
 epsilon, for the atom pair. The equilibrium separation has units of Å 
and the well depth, epsilon, units of kcal mol<em class="Superscript">
-1</em>
. The integer exponents <em class="Emphasis">
n</em>
 and <em class="Emphasis">
m</em>
 must be specified too. Obviously, <em class="Emphasis">
n&nbsp;</em>
&nbsp;<em class="Emphasis">
m</em>
. (Units: Å; kcal mol<em class="Superscript">
-1</em>
; none; none, respectively).</p>
<p class="Keyword">
<a name="pgfId=11026">
 </a>
intelec</p>
<p class="Body">
<a name="pgfId=11027">
 </a>
<em class="Emphasis">
(Optional) </em>
Internal ligand electrostatic energies will be calculated; the products 
of the partial charges in each non-bonded atom pair are pre-calculated, 
and output. Note that this is only relevant for flexible ligands.</p>
</div>
<div>
<ol>
<ol>
<ol>
<h4 class="AppndixHeading2">
<a name="pgfId=11028">
 </a>
Parameters for simulated annealing searches</h4>
</ol>
</ol>
</ol>
<p class="Keyword">
<a name="pgfId=11029">
 </a>
rt0 &lt;float&gt;</p>
<p class="Body">
<a name="pgfId=11030">
 </a>
[500. cal mol<em class="Superscript">
-1</em>
].<br>
Initial "annealing temperature"; this is actually the absolute temperature multiplied by the gas constant <em class="Emphasis">
R</em>
. <em class="Emphasis">
R</em>
 = 8.314 J mol<em class="Superscript">
-1 </em>
K<em class="Superscript">
-1 </em>
= 1.987 cal mol<em class="Superscript">
-1 </em>
K<em class="Superscript">
-1</em>
. (Units: cal mol<em class="Superscript">
-1</em>
.)</p>
<p class="Keyword">
<a name="pgfId=11031">
 </a>
rtrf &lt;float&gt;</p>
<p class="Body">
<a name="pgfId=11032">
 </a>
Annealing temperature reduction factor, <em class="Emphasis">
g</em>
 [0.95 cycle<em class="Superscript">
-1</em>
]. See the equation at the bottom of page 5. At the end of each cycle, 
the annealing temperature is multiplied by this factor, to give that of 
the next cycle. This must be positive but &lt; 1 in order to cool the 
system. Gradual cooling is recommended, so as to avoid "<em class="Emphasis">
simulated quenching</em>
", which tends to trap systems into local minima.</p>
<p class="Keyword">
<a name="pgfId=11033">
 </a>
linear_schedule</p>
<p class="Keyword">
<a name="pgfId=11034">
 </a>
schedule_linear</p>
<p class="Keyword">
<a name="pgfId=11035">
 </a>
linsched</p>
<p class="Keyword">
<a name="pgfId=11036">
 </a>
schedlin</p>
<p class="Body">
<a name="pgfId=11037">
 </a>
These keywords are all synonymous, and instruct <strong class="Bold">
AutoDock</strong>
 to use a linear or <em class="Emphasis">
arithmetic</em>
 temperature reduction schedule during <em class="Emphasis">
Monte Carlo</em>
 simulated annealing. Unless this keyword is given, a <em class="Emphasis">
geometric</em>
 reduction schedule is used, according to the <code class="Courier-Bold">
rtrf</code>
 parameter just described. If the linear schedule is requested, then any <code class="Courier-Bold">
rtrf</code>
 parameters will be ignored. The first simulated annealing cycle is carried out at the annealing temperature <code class="Courier-Bold">
rt0</code>
.  At the end of each cycle, the temperature is reduced by (<code class="Courier-Bold">
rt0/cycles</code>
). The advantage of the linear schedule is that the system samples 
evenly across the temperature axis, which is vital in entropic 
calculations. Geometric temperature reduction schedules on the other 
hand, under-sample high temperatures and over-sample low temperatures.</p>
<p class="Keyword">
<a name="pgfId=11038">
 </a>
runs &lt;integer&gt;</p>
<p class="Body">
<a name="pgfId=11039">
 </a>
[10]<br>
Number of automated docking runs.</p>
<p class="Keyword">
<a name="pgfId=11040">
 </a>
cycles &lt;integer&gt;</p>
<p class="Body">
<a name="pgfId=11041">
 </a>
[50]<br>
Number of temperature reduction cycles.</p>
<p class="Keyword">
<a name="pgfId=11042">
 </a>
accs &lt;integer&gt;</p>
<p class="Body">
<a name="pgfId=11043">
 </a>
[100]<br>
Maximum number of accepted steps per cycle.</p>
<p class="Keyword">
<a name="pgfId=11044">
 </a>
rejs &lt;integer&gt;</p>
<p class="Body">
<a name="pgfId=11045">
 </a>
[100]<br>
Maximum number of rejected steps per cycle.</p>
<p class="Keyword">
<a name="pgfId=11046">
 </a>
select &lt;character&gt;</p>
<p class="Body">
<a name="pgfId=11047">
 </a>
[m]<br>
State selection flag. This character can be either <code class="Courier-Bold">
m</code>
 for the <em class="Emphasis">
minimum</em>
 state, or <code class="Courier-Bold">
l</code>
 for the <em class="Emphasis">
last</em>
 state found during each cycle, to begin the following cycle.</p>
<p class="Keyword">
<a name="pgfId=11048">
 </a>
trnrf &lt;float&gt;</p>
<p class="Body">
<a name="pgfId=11049">
 </a>
[1.0]<br>
Per-cycle reduction factor for translations.</p>
<p class="Keyword">
<a name="pgfId=11050">
 </a>
quarf &lt;float&gt;</p>
<p class="Body">
<a name="pgfId=11051">
 </a>
[1.0]<br>
Per-cycle reduction factor for quaternions.</p>
<p class="Keyword">
<a name="pgfId=11052">
 </a>
dihrf &lt;float&gt;</p>
<p class="Body">
<a name="pgfId=11053">
 </a>
Per-cycle reduction factor for dihedrals [1.].</p>
</div>
<div>
<ol>
<ol>
<ol>
<h4 class="AppndixHeading2">
<a name="pgfId=11054">
 </a>
Parameter to set the amount of output</h4>
</ol>
</ol>
</ol>
<p class="Keyword">
<a name="pgfId=11055">
 </a>
outlev &lt;integer&gt;</p>
<p class="Body">
<a name="pgfId=11056">
 </a>
[1]<br>
Diagnostic output level. For SA (simulated annealing): 0 = no output, 1 =
 minimal output, 2 = full state output at end of each cycle; 3 = 
detailed output for each step. For GA and GA-LS (genetic algorithm-local
 search): 0 = minimal output, 1 = write minimum, mean, and maximum of 
each state variable at the end of every generation. Use "outlev 1" for 
SA, and "outlev 0" for GA and GA-LS. If you use "outlev 1" with GA-LS, 
you will generate very large log files.</p>
</div>
<div>
<ol>
<ol>
<ol>
<h4 class="AppndixHeading2">
<a name="pgfId=11057">
 </a>
Parameters for trajectory output during SA dockings</h4>
</ol>
</ol>
</ol>
<p class="Keyword">
<a name="pgfId=11058">
 </a>
trjfrq &lt;integer&gt;</p>
<p class="Body">
<a name="pgfId=11059">
 </a>
[0]<br>
Output frequency, <em class="Emphasis">
n</em>
, for trajectory of ligand, in steps. If <em class="Emphasis">
n</em>
 = 0, then no trajectory states will be output; otherwise, every <em class="Emphasis">
n</em>
th state will be output. The <em class="Emphasis">
state</em>
 consists of 7 floats describing the <em class="Emphasis">
x,y,z</em>
 translation, the <em class="Emphasis">
x,y,z</em>
 components of the quaternion unit vector, the angle of rotation about 
the quaternion axis; and any remaining floats describing the torsions, 
in the same order as described in the input ligand PDBQ file).</p>
<p class="Keyword">
<a name="pgfId=11060">
 </a>
trjbeg &lt;integer&gt;</p>
<p class="Body">
<a name="pgfId=11061">
 </a>
[1]<br>
Begin sampling states for trajectory output at the cycle with this value.</p>
<p class="Keyword">
<a name="pgfId=11062">
 </a>
trjend &lt;integer&gt;</p>
<p class="Body">
<a name="pgfId=11063">
 </a>
[50]<br>
End trajectory output at this cycle.</p>
<p class="Keyword">
<a name="pgfId=11064">
 </a>
trjout &lt;string&gt;</p>
<p class="Body">
<a name="pgfId=11065">
 </a>
[lig.trj]<br>
Trajectory filename. AutoDock will write out state variables to this 
file every "trjfrq" steps. Use the "traj" command in AutoDock's command 
mode to convert this trajectory of state-variables into a series of PDB 
frames. The "traj" command is described in § "Using the Command Mode in 
AutoDock"; see also § "Trajectory Files".</p>
<p class="Keyword">
<a name="pgfId=11066">
 </a>
trjsel &lt;string&gt;</p>
<p class="Body">
<a name="pgfId=11067">
 </a>
[E]<br>
Trajectory output flag, can be either `<code class="Courier">
A</code>
' or `<code class="Courier">
E</code>
'; the former outputs only <em class="Emphasis">
accepted</em>
 steps, while the latter outputs <em class="Emphasis">
either</em>
 accepted or rejected steps.</p>
<p class="Keyword">
<a name="pgfId=11068">
 </a>
watch &lt;string&gt;</p>
<p class="Body">
<a name="pgfId=11069">
 </a>
(<em class="Emphasis">
Optional) </em>
Creates a "watch" file for real-time monitoring of an <em class="Emphasis">
in-progress</em>
 simulated annealing job. This works only if the "trjfrq" parameter is 
greater than zero. The watch file will be in PDB format, so give a 
".pdb" extension. This file has an exclusive lock placed on it, while 
AutoDock is writing to it. Once the file is closed, the file is 
unlocked. This can signal to a watching visualization program that the 
file is complete and can now be read in, for updating the displayed 
coordinates. This file is written at exactly the same time as the 
trajectory file is updated</p>
</div>
<div>
<ol>
<ol>
<ol>
<h4 class="AppndixHeading2">
<a name="pgfId=11070">
 </a>
Parameter for energies of atoms outside the grid</h4>
</ol>
</ol>
</ol>
<p class="Keyword">
<a name="pgfId=11071">
 </a>
extnrg &lt;float&gt;</p>
<p class="Body">
<a name="pgfId=11072">
 </a>
[1000.]<br>
External grid energy assigned to any atoms that stray outside the volume of the grid during a docking. Units: kcal mol-1.</p>
</div>
<div>
<ol>
<ol>
<ol>
<h4 class="AppndixHeading2">
<a name="pgfId=11073">
 </a>
Parameter for initializing the ligand in SA</h4>
</ol>
</ol>
</ol>
<p class="Keyword">
<a name="pgfId=11074">
 </a>
e0max &lt;float&gt; &lt;positive_integer&gt;</p>
<p class="Body">
<a name="pgfId=11075">
 </a>
[0., 10000]<br>
This is only used by the simulated annealing method. This keyword 
stipulates that the ligand's initial state cannot have an energy greater
 than the first value, nor can there be more than the second value's 
number of retries. Typical energy values range from 0 to 1000 kcal/mol. 
If the initial energy exceeds this value, a new random state is 
generated and tested. This process is iterated until the condition is 
satisfied. This can be particularly useful in preventing runs starting 
in exceptionally high energy regions. In such cases, the ligand can get 
trapped because it is unable to take a long enough translational jump. 
In those grids were the ligand is small enough to fit into the low 
energy regions with ease, there will not be many iterations before a 
favorable location is found. But in highly constrained grids, with large
 ligands, this initialization loop may run almost indefinitely.</p>
</div>
<div>
<ol>
<ol>
<ol>
<h4 class="AppndixHeading2">
<a name="pgfId=11076">
 </a>
Parameters for cluster analysis of docked conformations</h4>
</ol>
</ol>
</ol>
<p class="Keyword">
<a name="pgfId=11077">
 </a>
rmsref &lt;string&gt;</p>
<p class="Body">
<a name="pgfId=11078">
 </a>
The root mean square deviation (rmsd) of the docked conformations will 
be calculated with respect to the coordinates in the PDB or PDBQ file 
specified here. This is useful when the experimentally determined 
complex conformation of the ligand is known. The order of the atoms in 
this file must match that in the input PDBQ file given by the <code class="Courier-Bold">
move</code>
 command. These values of rmsd will be output in the last column of the 
final PDBQ records, after the clustering has been performed.</p>
<p class="Keyword">
<a name="pgfId=11079">
 </a>
rmstol &lt;float&gt;</p>
<p class="Body">
<a name="pgfId=11080">
 </a>
[0.5Å]<br>
rms deviation tolerance for cluster analysis or `structure binning' , 
carried out after multiple docking runs. If two conformations have an 
rms less than this tolerance, they will be placed in the same cluster. 
The structures are ranked by energy, as are the clusters. The lowest 
energy representative from each cluster is output in PDBQ format to the 
log file. To keep the ligand's residue number in the input PDBQ file, 
use the `<code class="Courier-Bold">
-k</code>
' flag; otherwise the clustered conformations are numbered incrementally from 1. (Units: Å).</p>
<p class="Keyword">
<a name="pgfId=11081">
 </a>
rmsnosym</p>
<p class="Body">
<a name="pgfId=11082">
 </a>
When more than one run is carried out in a given job, cluster analysis 
or `structure binning' will be performed, based on structural rms 
difference, ranking the resulting families of docked conformations in 
order of increasing energy.  The default method for structure binning 
allows for atom similarity, as in a tertiary-butyl which can be rotated 
by +/-120°, but in other cases it may be desirable to bypass this 
similar atom type checking and calculate the rms on a one-for-one basis.
 The symmetry checking algorithm scans all atoms in the reference 
structure, and selects the nearest atom of identical atom type to be 
added to the sum of squares of distances. This works well when the two 
conformations are very similar, but this assumption breaks down when the
 two conformations are translated significantly. Symmetry checking can 
be turned off using the <code class="Courier-Bold">
rmsnosym</code>
 command; omit this command if you still want symmetry checking.</p>
</div>
<div>
<ol>
<ol>
<ol>
<h4 class="AppndixHeading2">
<a name="pgfId=29886">
 </a>
Parameters for re-clustering the results of several jobs</h4>
</ol>
</ol>
</ol>
<p class="Keyword">
<a name="pgfId=11083">
 </a>
cluster &lt;string&gt;</p>
<p class="Body">
<a name="pgfId=11084">
 </a>
<em class="Emphasis">
(Clustering multi-job output only.)</em>
 <strong class="Bold">
AutoDock </strong>
will go into `cluster mode'.  Use this command only to perform cluster analysis on the combined output,<em class="Emphasis">
 &lt;PDBQfilename&gt;</em>
, of several jobs. This command can be very useful when many jobs have 
been distributed to several machines and run in `parallel'. The docking 
parameter file will need the following keywords: <code class="Courier-Bold">
rmstol</code>
 and <code class="Courier-Bold">
types</code>
; and optionally write_all_cluster_members and/or <code class="Courier-Bold">
rmsnosym</code>
. </p>
<p class="Body">
<a name="pgfId=29891">
 </a>
It is necessary to <code class="Courier">
grep</code>
 the USER lines along with the ATOM records, since <strong class="Bold">
AutoDock</strong>
 parses the these lines to determine what the energy of that particular 
conformation was. For more information, see the example DPF files given 
later.</p>
<p class="Keyword">
<a name="pgfId=11085">
 </a>
write_all_cluster_members</p>
<p class="Body">
<a name="pgfId=11086">
 </a>
<em class="Emphasis">
(Clustering multi-job output only.)</em>
 This command is used only with the <strong class="Bold">
cluster</strong>
 command, to write out all members of each cluster instead of just the 
lowest energy from each cluster. This affects the cluster analysis PDBQ 
output at the end of each job.</p>
</div>
<div>
<ol>
<ol>
<ol>
<h4 class="AppndixHeading2">
<a name="pgfId=11087">
 </a>
Parameters for genetic algorithm, Lamarckian GA and evolutionary programming searches</h4>
</ol>
</ol>
</ol>
<p class="Keyword">
<a name="pgfId=11090">
 </a>
ga_pop_size &lt;positive_integer&gt;</p>
<p class="Body">
<a name="pgfId=11091">
 </a>
[50]<br>
This is the number of individuals in the population.  Each individual is
 a coupling of a genotype and its associated phenotype.  Usually, this 
number is fixed throughout the run.  Typical values range from 50 to 
200.</p>
<p class="Keyword">
<a name="pgfId=11092">
 </a>
ga_num_evals &lt;positive_integer&gt;</p>
<p class="Body">
<a name="pgfId=11093">
 </a>
[250000]<br>
This is the maximum number of energy evaluations that a GA run should make.</p>
<p class="Keyword">
<a name="pgfId=11094">
 </a>
ga_num_generations &lt;positive_integer&gt;</p>
<p class="Body">
<a name="pgfId=11095">
 </a>
[27000]<br>
This is the maximum number of generations that a GA or LGA run should last.</p>
<p class="Keyword">
<a name="pgfId=11096">
 </a>
ga_elitism &lt;integer&gt;</p>
<p class="Body">
<a name="pgfId=11097">
 </a>
[1]<br>
This is used in the selection mechanism of the GA. This is the number of
 top individuals that are guaranteed to survive into the next 
generation.</p>
<p class="Keyword">
<a name="pgfId=11098">
 </a>
ga_mutation_rate &lt;float&gt; </p>
<p class="Body">
<a name="pgfId=11099">
 </a>
[0.02]<br>
This is a floating point number from 0 to 1, representing the 
probability that a particular gene is mutated.  This parameter is 
typically small.</p>
<p class="Keyword">
<a name="pgfId=11100">
 </a>
ga_crossover_rate &lt;float&gt;</p>
<p class="Body">
<a name="pgfId=11101">
 </a>
[0.80]<br>
This is a floating point number from 0 to 1 denoting the crossover rate.
  Crossover rate is the expected number of pairs in the population that 
will exchange genetic material. Setting this value to 0 turns the GA 
into the evolutionary programming (EP) method, but EP would probably 
require a concomitant increase in the ga_mutation_rate in order to be 
effective.</p>
<p class="Keyword">
<a name="pgfId=11102">
 </a>
ga_window_size &lt;positive_integer&gt;</p>
<p class="Body">
<a name="pgfId=11103">
 </a>
[10]<br>
This is the number of preceding generations to take into consideration 
when deciding the threshold for the worst individual in the current 
population.</p>
<p class="Keyword">
<a name="pgfId=11104">
 </a>
ga_cauchy_alpha &lt;float&gt;</p>
<p class="Body">
<a name="pgfId=11105">
 </a>
[0]</p>
<p class="Keyword">
<a name="pgfId=11106">
 </a>
ga_cauchy_beta &lt;float&gt;</p>
<p class="Body">
<a name="pgfId=11107">
 </a>
[1]<br>
These are floating point parameters used in the mutation of real number 
genes.  They correspond to the alpha and beta parameters in a Cauchy 
distribution.  Alpha roughly corresponds to the mean, and beta to 
something like the variance of the distribution.  It should be noted, 
though, that the Cauchy distribution doesn't have finite variance.  For 
the mutation of a real valued gene, a Cauchy deviate is generated and 
then added to the original value.</p>
</div>
<div>
<ol>
<ol>
<ol>
<h4 class="AppndixHeading2">
<a name="pgfId=29837">
 </a>
Command to set genetic algorithm parameters</h4>
</ol>
</ol>
</ol>
<p class="Keyword">
<a name="pgfId=29831">
 </a>
set_ga</p>
<p class="Body">
<a name="pgfId=29848">
 </a>
This command sets the global optimizer to be a genetic algorithm [GA]. 
This is required to perform a GA search. This passes any '<code class="Courier-Small">
ga_</code>
' parameters specified <strong class="Bold">
before</strong>
 this line to the global optimizer object. If  this command is omitted, or it is given before the '<code class="Courier-Small">
ga_</code>
' parameters, your choices will not take effect, and the default values for the optimizer will be used.</p>
<p class="Body">
<a name="pgfId=29903">
 </a>
To use the traditional genetic algorithm, do not specify the local search parameters, and do not use the "<code class="Courier-Small">
set_sw1</code>
" or "<code class="Courier-Small">
set_psw1</code>
" commands.</p>
<p class="Body">
<a name="pgfId=29873">
 </a>
To use the <strong class="Bold">
Lamarckian genetic algorithm</strong>
, you <strong class="Bold">
must</strong>
 also specify the parameters for local search, and then issue either the '<code class="Courier-Small">
set_sw1</code>
' or '<code class="Courier-Small">
set_psw1</code>
' command.  The former command uses the strict Solis and Wets local 
search algorithm, while the latter uses the pseudo-Solis and Wets 
algorithm: see earlier for details about how they differ.</p>
</div>
<div>
<ol>
<ol>
<ol>
<h4 class="AppndixHeading2">
<a name="pgfId=11108">
 </a>
Parameters for local search</h4>
</ol>
</ol>
</ol>
<p class="Keyword">
<a name="pgfId=11113">
 </a>
sw_max_its &lt;positive_integer&gt;</p>
<p class="Body">
<a name="pgfId=11114">
 </a>
[50]<br>
This is the maximum number of iterations that the local search procedure
 apply to the phenotype of any given individual.  This is an unsigned 
integer.  In Bill's experiments, he used a combination of iterations and
 function evaluations.  It seems to me, that a value around 30 should be
 fine.</p>
<p class="Keyword">
<a name="pgfId=11115">
 </a>
sw_max_succ &lt;positive_integer&gt;</p>
<p class="Body">
<a name="pgfId=11116">
 </a>
[4]<br>
This is the number of successes in a row before a change is made to the 
rho parameter in Solis &amp; Wets algorithms.  This is an unsigned 
integer and is typically around four.</p>
<p class="Keyword">
<a name="pgfId=11117">
 </a>
sw_max_fail &lt;positive_integer&gt;</p>
<p class="Body">
<a name="pgfId=11118">
 </a>
[4]<br>
This is the number of failures in a row before Solis &amp; Wets 
algorithms adjust rho.  This is an unsigned integer and is usually 
around four.</p>
<p class="Keyword">
<a name="pgfId=11119">
 </a>
sw_rho &lt;float&gt;</p>
<p class="Body">
<a name="pgfId=11120">
 </a>
[1.0]<br>
This is a parameter of the Solis &amp; Wets algorithms.  It defines the 
initial variance, and specifies the size of the local space to sample.</p>
<p class="Keyword">
<a name="pgfId=11121">
 </a>
sw_lb_rho &lt;float&gt;</p>
<p class="Body">
<a name="pgfId=11122">
 </a>
[0.01]<br>
This is the lower bound on rho, the variance for making changes to genes (<em class="Emphasis">
i.e.</em>
 translations, orientation and torsions).  rho can never be modified to a value smaller than "<code class="Courier-Small">
sw_lb_rho</code>
".</p>
<p class="Keyword">
<a name="pgfId=11123">
 </a>
ls_search_freq &lt;float&gt;</p>
<p class="Body">
<a name="pgfId=29811">
 </a>
[0.06]</p>
<p class="Body">
<a name="pgfId=11124">
 </a>
This is the probability of any particular phenotype being subjected to local search.</p>
</div>
<div>
<ol>
<ol>
<ol>
<h4 class="AppndixHeading2">
<a name="pgfId=29785">
 </a>
Commands to choose and set the local search method</h4>
</ol>
</ol>
</ol>
<p class="Body">
<a name="pgfId=29796">
 </a>
Both of these commands, '<code class="Courier-Small">
set_sw1</code>
' and '<code class="Courier-Small">
set_psw1</code>
', pass any '<code class="Courier-Small">
sw_</code>
' parameters set before this line to the local searcher. If you forget to use this command, or give it before the '<code class="Courier-Small">
sw_</code>
' keywords, your choices will not take effect, and the default values for the optimizer will be used.</p>
<p class="Keyword">
<a name="pgfId=29775">
 </a>
set_sw1</p>
<p class="Body">
<a name="pgfId=29776">
 </a>
Instructs AutoDock to use the classical Solis and Wets local searcher, 
using the method of uniform variances for changes in translations, 
orientations and torsions.</p>
<p class="Keyword">
<a name="pgfId=29777">
 </a>
set_psw1</p>
<p class="Body">
<a name="pgfId=29778">
 </a>
Instructs AutoDock to use the pseudo-Solis and Wets local searcher. This
 method maintains the relative proportions of variances for the 
translations in Å and the rotations in radians. These are typically 0.2 Å
 and 0.087 radians to start with, so the variance for translations will 
always be about 2.3 times larger than that for the rotations (<em class="Emphasis">
i.e.</em>
 orientation and torsions).</p>
</div>
<div>
<ol>
<ol>
<ol>
<h4 class="AppndixHeading2">
<a name="pgfId=11125">
 </a>
Commands to perform automated docking</h4>
</ol>
</ol>
</ol>
<p class="Keyword">
<a name="pgfId=11126">
 </a>
simanneal</p>
<p class="Body">
<a name="pgfId=11127">
 </a>
This command instructs AutoDock to do the specifed number of docking 
runs using the simulated annealing (SA) search engine. This uses the 
value set by the "<code class="Courier-Small">
runs</code>
" keyword as the number of SA docking runs to carry out. All relevant 
parameters for the simulated annealing job must be set first. These are 
indicated above by <strong class="Bold">
[SA] </strong>
in each keyword description.</p>
<p class="Keyword">
<a name="pgfId=11128">
 </a>
do_local_only &lt;integer&gt;</p>
<p class="Body">
<a name="pgfId=11129">
 </a>
This keyword instructs AutoDock to carry out only the local search of a 
global-local search; the genetic algorithm parameters are ignored, with 
the exception of the population size. This is an ideal way of carrying 
out a minimization using the same force field as is used during the 
dockings. The "<code class="Courier-Small">
ga_run</code>
" keyword should not be given. The number after the keyword determines how many dockings will be performed.</p>
<p class="Keyword">
<a name="pgfId=11130">
 </a>
do_global_only &lt;integer&gt;</p>
<p class="Body">
<a name="pgfId=11131">
 </a>
This keyword instructs AutoDock to carry out dockings using only a 
global search, i.e. the traditional genetic algorithm. The local search 
parameters are ignored. The "<code class="Courier-Small">
ga_run</code>
" keyword should not be given. The number after the keyword determines how many dockings will be performed.</p>
<p class="Keyword">
<a name="pgfId=11132">
 </a>
ga_run &lt;integer&gt;</p>
<p class="Body">
<a name="pgfId=11134">
 </a>
[10]<br>
This command invokes the new hybrid, Lamarckian genetic algorithm search
 engine, and performs the requested number of dockings. All appropriate 
parameters must be set first: these are listed above by "<code class="Courier-Small">
ga_</code>
".</p>
</div>
<div>
<ol>
<ol>
<ol>
<h4 class="AppndixHeading2">
<a name="pgfId=11135">
 </a>
Command to perform clustering of docked conformations</h4>
</ol>
</ol>
</ol>
<p class="Keyword">
<a name="pgfId=11136">
 </a>
analysis</p>
<p class="Body">
<a name="pgfId=11137">
 </a>
This performs a cluster analysis on results of a docking, and outputs 
the results to the log file. The docked conformations are sorted in 
order of increasing energy, then compared by root mean square deviation.
 If the conformer is within the "rmstol" threshold, it is placed into 
the same cluster. A histogram is printed showing the number in each 
cluster, and if more than one member, the cluster's mean energy. 
Furthermore, a table is printed to the docking log file of cluster rmsd 
and reference rmsd values.</p>
</div>
</div>
<hr>
<div class="footnotes">
<div class="footnote">
<p class="Footnote">
<span class="footnoteNumber">
1.</span>
<a name="pgfId=30081">
 </a>
Stouten, P. F. W., Frömmel, C., Nakamura, H. and Sander, C. (1993). "An 
effective solvation term based on atomic occupancies for use in protein 
simulations", <em class="Emphasis">
Molecular Simulations</em>
, <strong class="Bold">
10</strong>
, 97-120.</p>
</div>
<div class="footnote">
<p class="Footnote">
<span class="footnoteNumber">
2.</span>
<a name="pgfId=10758">
 </a>
"AVS" stands for "Application Visualization System"; AVS is a trademark 
of Advanced Visual Systems Inc., 300 Fifth Avenue, Waltham, MA 02154.</p>
</div>
<div class="footnote">
<p class="Footnote">
<span class="footnoteNumber">
3.</span>
<a name="pgfId=10783">
 </a>
This grid map is not used in AutoDock 3.0; its utility is under investigation, and may be included in a later version.</p>
</div>
</div>


</body></html>
